webhooks.md

```diff
@@ -0,0 +1,105 @@
+# Webhooks
+
+When an action of particular type is performed in the system, a message is sent to the configured webhook, using HTTP POST and JSON body.
+
+## Message
+
+```javascript
+{
+    "Action": "IntegrationCreated",
+    "Data": {
+        "Enterprise": {
+            "Id": "8865aa96-f62d-4f9b-a912-ab2100f60f42",
+            "Name": "Sample Chain Hotel 1"
+        },
+        "Requestor": null,
+        "AccessToken":"9E5E84E9974D4F169662AB2200F27CB1-00B343A0DDA725CACAC028E38E3EABF",
+        "CreatedUtc": "2019-12-13T14:42:52Z",
+        "IsEnabled": true,
+        "Integration": {
+            "Id": "9e5e84e9-974d-4f16-9662-ab2200f27cb1",
+            "Name": "WebhookTEST"
+        }
+    }
+}
+
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Action` | string [Webhook action](#webhook-action) | required | Type of the action. |
+| `Data` | object [Data](#integration-created-data) | required | Webhook data. |
```

Comment:
Here it should link to section "Webhook data" and there it should describe that the type depends on the action. Maybe check how device command data are done in documentation for operations. That is exactly same case, so let's make it same structurally.